api/data_explorer/test/test_dataset_controller.py

```diff
@@ -17,7 +17,15 @@ def test_dataset_get(self):
         """Test case for dataset_get"""
         response = self.client.get('/dataset')
         self.assert200(response)
-        self.assertEquals(dict(name='Test data'), response.json)
+        self.assertEquals(
+            dict(
+                name='Test data',
+                tableNames=[
+                    "test_project1.test_dataset1.test_table1",
```

Comment:
super nit - These numbers are kind of random. Can there just a simple pattern, like
```
1 1 1
2 2 2
```

Also we don't need 4 table names, I think 2 is sufficient. I like to have test data be the minimal needed for testing.

Reply:
The numbers are incremental starting from 1, what seems random about that? And I chose these numbers because they exercise the 4 cases we'd expect to see:
* Same project/dataset/table
* Same project different dataset
* Same project/dataset different table
* Different project